data-prepper-api/src/main/java/org/opensearch/dataprepper/model/event/DefaultEventMetadata.java

```diff
@@ -68,6 +68,25 @@ public Map<String, Object> getAttributes() {
         return attributes;
     }
 
+    @Override
+    public Object getAttribute(final String attributeKey) {
+        String key = (attributeKey.charAt(0) == '/') ? attributeKey.substring(1) : attributeKey;
```

Comment:
This won't cover all the scenarios that JSON Pointer supports in event.

I think we have two options to consider here:

1. Implement full support here.
2. Only support the "simple" attributes to start. ie. `getMetadata(key)`. Don't support anything that looks like a JSON Pointer.

Thoughts?

Reply:
Also, if we do approach 1, we may want to consider if we can do this in JacksonEvent to try to improve performance.

Reply:
@dlvenable I am OK with supporting simple attributes for now.

Reply:
Modified to support simple attributes.